google-cloud-speech/acceptance/speech/async_test.rb

```diff
@@ -166,4 +166,21 @@
     results.first.confidence.must_be_close_to 0.9, 0.1
     results.first.alternatives.must_be :empty?
   end
+
+  it "recognizes audio with words" do
+    op = speech.process filepath, encoding: :linear16, language: "en-US", sample_rate: 16000, words: true
```

Comment:
We usually use "timestamp" to mean a full date/time value, not a Duration. 

Our current direction of favoring Ruby norms over the API names/structures was initiated by JJ. I assume he is still in favor of it.